examples/style_sheets/style_sheets_reference.py

```diff
@@ -108,17 +113,28 @@ def plot_figure(style_label=""):
     # double the width and halve the height. NB: use relative changes because
     # some styles may have a figure size different from the default one.
     (fig_width, fig_height) = plt.rcParams['figure.figsize']
-    fig_size = [fig_width * 2, fig_height / 2]
+    fig_size = [fig_width * 2, fig_height / 1.75]
 
     fig, axs = plt.subplots(ncols=6, nrows=1, num=style_label,
                             figsize=fig_size, squeeze=True)
-    axs[0].set_ylabel(style_label)
+
+    # make a suptitle, in the same style for all subfigures,
+    # except those with dark backgrounds, which get a lighter
+    # color:
+    col = np.array([19, 6, 84])/256
+    back = mcolors.rgb_to_hsv(
+        mcolors.to_rgb(plt.rcParams['figure.facecolor']))[2]
+    if back < 0.5:
+        col = [0.8, 0.8, 1]
+    fig.suptitle(style_label, x=0.01, fontsize=14, ha='left',
```

Comment:
This looks good, but many times the fontsizes are are different?  https://65288-1385122-gh.circle-artifacts.com/0/doc/build/html/gallery/style_sheets/style_sheets_reference.html  It think this is `tight_layout` making the axes smaller or bigger and then css making all the figures the same size.   I think if you switch to constrained_layout this will be better (tight_layout doesn't work with suptitles)

Reply:
Ok, let's try with constrained layout.

Reply:
That didn't work either.  I don't really understand why the fontsize is not being respected...

Reply:
The main "problem" with the fontsize seems to be that the different styles define different figure sizes. E.g. compare 

[default](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/forks/517176bdf7d05e9e46d47076/110003219/61acbbba9a352657477f5e61-0-build/artifacts/doc/build/html/_images/sphx_glr_style_sheets_reference_001_2_0x.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20211216T211759Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20211216%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=9cb516f7ce92c781fb52afbdfb8392a887a84a0ae193b6c57eafd566fca8c35e)

and 

[seaborn-poster](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/forks/517176bdf7d05e9e46d47076/110003219/61acbbba9a352657477f5e61-0-build/artifacts/doc/build/html/_images/sphx_glr_style_sheets_reference_021_2_0x.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20211216T211824Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20211216%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=3f5fea7ad0d837ea72d2e7cfa486615ca23de0f23b2c6b559f19376467342177)

The fonts have actually the same size, but the poster is much larger, and thus the png is scaled down to fit in the available width in the theme.

The alternative would be to use an absolute figure size instead of a one relative `rcParams['figure.size']`. Not sure which is better.

Reply:
Oh weird. That hadn't occurred to me. I'd just make the figure a fixed size.  That's essentially what we did w the subfigure version and I think that looked ok?

Reply:
Done: 

It's better to override absolute sizes in the style sheet. Otherwise,
larger figure images are scaled down due to the limited available width
in the html theme. If not, we still don't get a size relation, but we
also have different magnifications due to the image scaling.

Notes:

- The width (7.4 -> 740px, assuming 100 dpi) is chosen to fit in the
  HTML without scaling.
- The size is chosen to approximately keep the aspect ratio as before.
- The images are now significantly smaller than previously (all former
  images were downscaled). We'll have to verify that the images still
  look good given a lot of information and not much space. If not, we
  need to add a factor.
- All themes use figure.dpi=100, except "classic", which uses 80. Not
  correcting for that is intentional. IMHO "classic" should come out
  smaller to highlight the change in dpi.